contracts/Campaign.sol

```diff
@@ -136,4 +137,11 @@ contract Campaign {
         }
         */
     }
+
+    function returnRemainingBalance() public campaignFactoryOnly distrustOnly {
+        address payable campgnFactory = address(uint160(campaignFactory));
+        uint256 remainingBalance =  address(this).balance;
+        campgnFactory.transfer(remainingBalance);
+        emit hasReturnedBalance(address(this), remainingBalance);
```

Comment:
Is this method necessary? Afraid it might confuse the marker.

```suggestion
        address payable campaignFactory = address(uint160(campaignFactory));
        uint256 remainingBalance =  address(this).balance;
        campaignFactory.transfer(remainingBalance);
        emit hasReturnedBalance(address(this), remainingBalance);
```

Reply:
Thanks for the suggestion. This line will be removed following latest PR #10 as it includes conversion of campaignFactory and testCampaignFactory to be payable. Hence, should not need explicit conversion again here. 

The Withdraw function will also be standardized accordingly.